src/OSPSuite.Presentation/Presenters/Importer/ImporterPresenter.cs

```diff
@@ -404,6 +404,8 @@ public void LoadConfigurationWithoutImporting()
          if (confirmDroppingOfLoadedSheets())
                return;
 
+         onResetMappingBasedOnCurrentSheet(this, null);
```

Comment:
onResetMappingBasedOnCurrentSheet does not use either e nor null. Can I suggest to remove the argument in this mehtod, call in resetMappingBasedOnCurrentSheet and in the event registration, do (o, e)=> eesetMappingBasedOnCurrentSheet()

Reply:
You are right